docs/how-to/heterogeneous_matches.rst

```diff
@@ -0,0 +1,65 @@
+.. _heterogeneous-matches:
+
+Heterogeneous Matches
+=====================
+
+Axelrod Matches are homogeneous by nature but can be extended to utilize additional attributes of heterogeneous players. 
+This tutorial indicates how the Axelrod :code:`Match` class can be manipulated in order to play heterogeneous tournaments and Moran processes using country mass as a score modifier.
```

Comment:
I am not sure I understand this sentence. Are you trying to say that the masses now influence the score of each player?

Reply:
Sorry if it's worded unclearly! I was trying to say that the masses could for example be used to modify Match elements such as the final score.